homeassistant/components/yeelight/light.py

```diff
@@ -281,6 +300,42 @@ def light_type(self):
         """Return light type."""
         return self._light_type
 
+    @property
+    def hs_color(self) -> tuple:
+        """Return the color property."""
+        return self._hs
+
+    # F821: https://github.com/PyCQA/pyflakes/issues/373
+    @property
+    def _bulb(self) -> 'yeelight.Bulb':  # noqa: F821
+        return self.device.bulb
+
+    @property
+    def _properties(self) -> dict:
```

Comment:
I don't think `dict` is a correct type hint, is it? Or is using `Dict` from `typing` only necessary if you want to describe the contained types?

Reply:
I believe dict type hint was introducend here by you https://github.com/rytilahti/home-assistant/commit/cf5d9cec0d062b8883d1cc085c2463b5d8c2c2dd#diff-1395165791bf7671502a6e2e7bd7d4cbR173 . I'm not sure whats best here. Whats wrong with dict ? Maybe we can skip type hint ?

Reply:
I'm not sure if there's something wrong with just using dict, the typing module has `Dict`, but maybe it is just for those cases where you want to define types of its keys and values. So I suppose this is fine..